core/src/main/java/com/linecorp/armeria/client/retry/BackoffSpec.java

```diff
@@ -275,6 +299,8 @@ Backoff build() {
             backoff = Backoff.fixed(fixedDelayMillis);
         } else if (baseOption == BaseOption.random) {
             backoff = Backoff.random(randomMinDelayMillis, randomMaxDelayMillis);
+        } else if (baseOption == BaseOption.fibonacci) {
+            backoff = Backoff.fibonacci(randomMinDelayMillis, randomMaxDelayMillis);
```

Comment:
This should be
`backoff = Backoff.fibonacci(initialDelayMillis, maxDelayMillis);`

Reply:
Well spotted, I completely missed it. 🙇🏻‍♂️